src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String SYMBOL_SEPARATOR = ",";
+    private static final int NUMBER_OPERATION_TYPE_COLUMN = 0;
+    private static final int NUMBER_AMOUNT_COLUMN = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileContent = getStringFromFile(fromFileName);
```

Comment:
this method is already incorrect - it does too many things

you should have separate methods for reading data, calculating report and writing date